README.md

```diff
@@ -18,7 +18,7 @@ The RHTAS Ansible collection deploys the following RHTAS components:
 
 * [Rekor](https://docs.sigstore.dev/rekor/overview)
   * [Trillian database](https://github.com/google/trillian)
-  * Optional: A self-managed MariaDB instance, and a Redis instance.
+  * [Optional: A self-managed MariaDB instance, and a Redis instance.](#configuring-a-user-provisioned-mariadb-and-redis-instance-for-the-rhtas-ansible-collection)
```

Comment:
Can we add a note here that this is highly recommended for production deployments, because otherwise people would have to manage the services that we provisioned, handle their data backups etc.